docs/runner/components/events_create.rst

```diff
@@ -60,32 +60,6 @@ An example when opening a door in Paris:
         confirm: true
 
 
-An example when releasing a software package as an sbom:
```

Comment:
we still want this don't we?

Reply:
No - this yaml has an 'sbom:' dict which means that the sbom is created b=y accessing sbom hub whose endpoint has now disappeared.